src/template-analyzer.ts

```diff
@@ -164,6 +168,8 @@ export class TemplateAnalyzer {
     }
 
     const loc = element.sourceCodeLocation.attrs[originalAttr];
+    if (!loc) return null;
```

Comment:
there's meant to be a `braces` lint rule enabled, surprised this one didn't get flagged:

```ts
if (!loc) {
  return null;
}
```